test/runtests.jl

```diff
@@ -17,6 +17,24 @@ test_show(lm1)
 @test_approx_eq vcov(lm1) Σ
 @test_approx_eq cor(lm1.model) diagm(diag(Σ))^(-1/2)*Σ*diagm(diag(Σ))^(-1/2)
 
+#1).
```

Comment:
This line isn't very useful, it's even confusing since the object is called `gm`, not `gm1`.